test_cases/README.md

```diff
@@ -32,6 +32,13 @@ stubs where decisions have been taken that might be slightly unusual. These
 test cases serve a different purpose: to check that type checkers do not emit
 false-positive errors for idiomatic usage of these classes.
 
+## Running the tests
+
+To run the tests in this directory, run `python tests/regr_test.py stdlib`
+from the root of the typeshed repository. This assumes that the development
+environment has been set up as described in the [CONTRIBUTING.md](../CONTRIBUTING.md)
+document.
```

Comment:
These instructions are mypy-specific, but we also run pyright on these test cases in CI. Should we also add some instructions for pyright users?

I think the command you'd need to run would be `pyright --project pyrightconfig.testcases.json test_cases`, but I don't actually have pyright installed locally, so I can't verify that.

Reply:
Let's go with these instructions for now. It would be useful to add a small stub script in tests, though, that call pyright with the correct paramters.

Reply:
How about just this for now, then?

```suggestion
To verify the test cases in this directory with mypy, run `python tests/regr_test.py stdlib`
from the root of the typeshed repository. This assumes that the development
environment has been set up as described in the [CONTRIBUTING.md](../CONTRIBUTING.md)
document.
```

My hesitation with the current wording is that part of the point of these test cases is that they're supposed to be typechecker-agnostic, but the script is mypy-specific. (In theory we could start running pytype/pyre on these test cases at some point, as well as mypy and pyright.)